src/js/views/maps/viewfinder/SearchView.js

```diff
@@ -0,0 +1,234 @@
+'use strict';
+
+define(
+  [
+    'underscore',
+    'backbone',
+    'text!templates/maps/viewfinder/viewfinder-search.html',
+    'views/maps/viewfinder/PredictionsListView',
+    'models/maps/viewfinder/ViewfinderModel',
+    "views/maps/SearchInputView",
+  ],
+  (
+    _,
+    Backbone,
+    Template,
+    PredictionsListView,
+    ViewfinderModel,
+    SearchInputView,
+  ) => {
+    // The base classname to use for this View's template elements.
+    const BASE_CLASS = 'viewfinder';
+    // The HTML classes to use for this view's HTML elements.
+    const CLASS_NAMES = {
+      predictions: `${BASE_CLASS}__predictions`,
+      searchInput: `${BASE_CLASS}__search-input`,
+    };
+
+    /**
+     * @class SearchView
+     * @classdesc SearchView allows a user to search for
+     * a latitude and longitude in the map view, and find suggestions
+     * for places related to their search terms.
+     * This view requires a Google Maps API key in order to function properly,
+     * and must have the Geocoding API and Places API enabled.
+     * @classcategory Views/Maps
+     * @name SearchView
+     * @extends Backbone.View
+     * @screenshot views/maps/viewfinder/SearchView.png
+     * @since x.x.x
+     * @constructs SearchView
+     */
+    var SearchView = Backbone.View.extend({
+      /**
+       * The type of View this is
+       * @type {string}
+       */
+      type: 'SearchView',
+
+      /** @inheritdoc */
+      className: BASE_CLASS,
+
+      /** 
+       * Values meant to be used by the rendered HTML template.
+       */
+      templateVars: {
+        classNames: CLASS_NAMES,
+      },
+
+      /**
+       * @typedef {Object} SearchViewOptions
+       * @property {Map} The Map model associated with this view allowing control
+       * of panning to different locations on the map. 
+       */
+      initialize({ viewfinderModel }) {
```

Comment:
Can/should we use a new ViewfinderModel if one isn't passed to the view?

Reply:
I don't think it would be particularly useful, since the ViewfinderModel needs to have the Map model that corresponds to the Cesium application.  What do you think?

Reply:
Ah I see, then agree it's not useful!